.github/workflows/github_actions_build.yml

```diff
@@ -91,4 +91,6 @@ jobs:
         export PATH="$LLVMBASE/bin:$PATH"
 
         cd ${{ matrix.ENV }}
+        make demos ${{ matrix.arch }}=Y
+        make clean
```

Comment:
What do you think about using build directories instead? This would also allow to provide these as separate artifacts for downloads.